### PR TITLE
Home page

### LINKED ISSUE #165: create home page

### PR DESCRIPTION
**Pull request by blukai**
fixes #165

### CODE REVIEW

**Comment by howardchung** [comment]
sync with master and fix CI errors, otherwise LGTM

**Comment by howardchung** [comment]
@albertcui should we feature our sponsors on the home page?

**Comment by blukai** [comment]
image

File: src/components/Footer/Links.jsx [hunk 1]
```diff
@@ -21,6 +21,12 @@ export default () => (
       >
         <IconSteam style={{ width: 14, height: 14 }} />
       </IconButton>
+      {location.location === '/' &&
```
Comment (howardchung) [inline src/components/Footer/Links.jsx]: Honestly, I don't mind just putting the attribution on every page if it makes things simpler :)

File: src/components/Home/Buttons.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            <span>
+              {strings.parse_desc}
+            </span>
+          </span>
+        }
+        containerElement={<Link to="/request" />}
+      />
+    </div>
+    <div>
+      <FlatButton
```
Comment (howardchung) [inline src/components/Home/Buttons.jsx]: I'd actually prefer to have a separate sponsors section on the page.  Jist will have the first position as our longest-tenured sponsor.

I like how Vue does it on their page: https://vuejs.org/

@albertcui what do you think?

File: src/components/Home/Home.jsx [hunk 3]
```diff
@@ -1,8 +1,24 @@
...
+import Buttons from './Buttons';
+import Why from './Why';
+import styles from './Home.css';
 
 const Home = () => (
-  <div>This is a home page!</div>
+  <div>
+    <div className={styles.HeadContainer}>
+      <div className={styles.headline}>
+        {strings.app_name}
```
Comment (howardchung) [inline src/components/Home/Home.jsx]: Should we style it `<OPENDOTA/> like we do in AppLogo?

File: src/components/Home/Home.jsx [hunk 4]
```diff
@@ -1,8 +1,24 @@
...
+      </div>
+      <div className={styles.description}>
+        {strings.app_description}
+      </div>
+      <Buttons />
+    </div>
+    <Why />
+    {/* Add some screenshots with description */}
```
Comment (howardchung) [inline src/components/Home/Home.jsx]: mark this //TODO and/or create an issue

File: src/components/Footer/Links.jsx (hunk 1 above)
Comment (howardchung) [inline src/components/Footer/Links.jsx]: Alternatively, you could put the attribution with the image like we currently do for the carry page:
https://www.opendota.com/carry

File: src/components/Home/Home.jsx (hunk 3 above)
Comment (blukai) [inline src/components/Home/Home.jsx]: nope 😄

File: src/components/Footer/Links.jsx (hunk 1 above)
Comment (blukai) [inline src/components/Footer/Links.jsx]: It looks not nice on pic, I'll put it on bottom of the home page

File: src/components/Home/Buttons.jsx (hunk 2 above)
Comment (blukai) [inline src/components/Home/Buttons.jsx]: I guess sponsors should give us images that they wanna see on home page.

File: src/components/Home/Home.jsx (hunk 4 above)
Comment (blukai) [inline src/components/Home/Home.jsx]: #237

File: src/components/Home/Buttons.jsx (hunk 2 above)
Comment (howardchung) [inline src/components/Home/Buttons.jsx]: I think there are logos/images for our sponsors in the public/images directory of odota/core.

**Comment by nicholashh** [comment]
SEXY AF -signoff-

**Pull request merged**